### DISCUSSION TITLE
Push to Cachix?

### DISCUSSION THREAD

**Discussion by r-k-b** [nix-community/browser-previews#4] [Ideas]
This might qualify for Cachix's free tier?

The build won't get much faster, but it'll be good practice for pushing builds to other (private) caches.

**Comment by JohnRTitor** [comment]
I think so. But even in Nixpkgs, unfree packages are not cached at all, as they are not redistributable (the licence doesn't permit it).
Unfree packages are always fetched from the source and built locally, though the overhead for binaries like this is low since they need to be only patched.
